scripts/configs/offline_config.yaml

```diff
@@ -0,0 +1,36 @@
+# PG4 experiments.
+---
+APPROACHES:
+  pg3:
+    NAME: "pg3"
+    FLAGS: {}
+  plan-only:
+    NAME: "oracle"
+    FLAGS: {}
+    
+ENVS:
+  deliver:
+    NAME: "pddl_easy_delivery_procedural_tasks"
+    FLAGS:
+      explore_random_options:
```

Comment:
these flags look like they are for the online learning experiments, not the offline learning ones -- please remove and update. also for any flags that are not specific to an environment, let's put them in the general `FLAGS` section below (currently on line 30)